### ISSUE TITLE
Cannot Import Servers "Unable to load the specified file."

### ISSUE THREAD

**Issue by OG-BT** [pgadmin-org/pgadmin4#5783]
**Describe the bug**
After exporting my servers within pgAdmin I cannot import them. I have the JSON file and can read it without issue.

**To Reproduce**

1. Attempt to import server after making a export from pgAdmin

**Expected behavior**

To continue to the part where I select the servers. But I get greeted with a "Unable to load the specified file." error at the bottom

**Error message**

"Unable to load the specified file."

**Screenshots**

![Screenshot 2023-01-25 at 2 21 02 PM](https://user-images.githubusercontent.com/123590295/214665686-4149edd9-45f2-450d-8f0a-9c420150939f.png)

**Desktop (please complete the following information):**
 - OS: macOS Ventura 13.1
 - Version: 13.1
 - Mode: Desktop

**Comment by akshay-joshi** [comment]
@OG-BT 

Are there any special characters in the path? In the screenshot, the path seems to have some blank spaces.

**Comment by nikhil-mohite** [comment]
@OG-BT ,

This is working for us, can you please provide the JSON file if possible?

**Comment by OG-BT** [comment]
@akshay-joshi The folder structure and filename have some spaces. I tried removing those and it still errored out. 

<img width="1543" alt="Screenshot 2023-01-27 at 10 00 59 AM" src="https://user-images.githubusercontent.com/123590295/215117871-ea798437-0170-4f48-99c0-05cf5fe9d82b.png">

@nikhil-mohite It's quite an extensive database, it contains all the data for our Davinci Resolve video project files.
I uploaded it to Google Drive: https://drive.google.com/file/d/1Lr8W6O30-UtLHQ_rKM37sBZRzORqJm2F/view?usp=sharing
